docs/section22/Rule22-42.md

```diff
@@ -0,0 +1,99 @@
+        - get the chiller rated capacity: `rated_capacity = chiller.rated_capacity`
+        - convert the chiller rated capacity from watts to tons: `chiller_capacity_tons = rated_capacity * 0.000284345`
+        - check if the compressor type is CENTRIFUGAL: `if chiller.compressor_type == "CENTRIFUGAL":`
+            - if the capacity is less than 150 tons, the category is Z: `if chiller_capacity_tons < 150: curve_set = "Z"`
+            - else if the capacity is greater than or equal to 300 tons, the category is AB: `elif chiller_capacity_tons >= 300: curve_set = "AB"`
+            - otherwise, the capacity is between 150 and 300, so the curves set is AA: `else: curve_set = "AA"`
+        - otherwise, check if the compressor type is POSITIVE_DISPLACEMENT or SCROLL: `if chiller.compressor_type in ["POSITIVE_DISPLACEMENT","SCROLL"]:`
+            - if the capacity is less than 150 tons, the category is Z: `if chiller_capacity_tons < 150: curve_set = "Z"`
+            - else if the capacity is greater than or equal to 300 tons, the category is AA: `elif chiller_capacity_tons >= 300: curve_set = "AA"`
+            - otherwise, the capacity is between 150 and 300, so the curves set is AB: `else: curve_set = "AB"`
+        - continue if the curve set is not "NONE" - the curve set could be none if the chiller.compressor_type is not one of the recognized types for Appendix G: `if curve_set <> "NONE":`
+            - calculate the chiller rated power by dividing the chiller rated capacity by the chiller full load efficiency (cop): `rated_power = chiller.rated_capacity / chiller.full_load_efficiency`
+            - create a list of the expected capacity validation points - this is in 25% intervals, starting with 0.25 to 1: `expected_validation_plr = [0.25,0.5,0.75,1]`
+            - create a list of the expected temperatures for the chilled water temperature for the validation calculations - expected_chwt_temps - units are (F).  The values are selected to capture the minimum, maximum, rating condition and some points in between: `expected_chwt_temps = [39,45,50,55]`
+            - create a list of the expected entering condenser temperatures for the condenser water temperature for the validation calculations - expected_ecwt_temps - units are (F).  The values are selected to capture the minimum, maximum, rating condition and some points in between: `expected_ecwt_temps = [60,104,85,72.5,97.5]`
+
+            - get the coefficients for the EIR-f-T curve (IP units) using a table lookup for table J-6.  This assumes that the lookup function will return a list with the coefficients in order (C1,C2,...): `eir_f_t_coefficients = table_J_2_lookup(curve_set,"EIR-f-T")`
+            - get the coefficients for the CAP-f-T curve (IP units) using a table lookup for table J-6.  This assumes that the lookup function will return a list with the coefficients in order (C1,C2,...): `cap_f_t_coefficients = table_J_2_lookup(curve_set,"Cap-f-T")`
```

Comment:
@weilixu do we need to create these lookup tables or do they exist for OS Standards?